docs/source/nlp/tutorials/quickstarts/plot_multi_label_classification.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+NLP Multi Label Classification Quickstart
+*****************************************
+
+In this quickstart guide, we will go over using the deepchecks NLP package to analyze and evaluate a text
```

Comment:
Would link to the multiclass quickstart somewhere here as it has a bit more stuff. not a must